File: Java/Estrutura condicional/Operadora.java
import java.util.Locale;
import java.util.Scanner;

public class Operadora {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int minutos;
        double valor;

        System.out.print("Digite a quantidade de minutos: ");
        minutos = sc.nextInt();

        if (minutos <= 100) {
            valor = 50.00;
        } else {
            valor = (minutos - 100) * 2.0 + 50.00;
        }

        System.out.println("Valo a pagar: R$ " + String.format("%.2f", valor));



        sc.close();

    }
}

File: Python/Estrutura Condicional/Operadora.py
minutos: int
valor: float

minutos = int(input("Digite a quantidade de minutos: "))

if minutos <= 100:
    valor = 50.00
else:
    valor = 50.00 + ((minutos - 100) * 2)


print(f"Valor a pagar: R$ {valor:.2f}")



